trigger_status.yml

```diff
@@ -0,0 +1,4 @@
+Opportunity:CRLP_Rollup_TDTM: true
```

Comment:
This file needs to be removed from the PR

Reply:
Agreed, thanks for pointing it out